consensus/polybft/consensus_runtime.go

```diff
@@ -309,21 +309,6 @@ func (c *consensusRuntime) populateFsmIfBridgeEnabled(
 		if err := c.state.cleanCommitments(nextStateSyncExecutionIdx); err != nil {
 			return fmt.Errorf("cannot clean commitments: %w", err)
 		}
-
-		nonExecutedCommitments, err := c.state.getNonExecutedCommitments(nextStateSyncExecutionIdx)
-		if err != nil {
-			return fmt.Errorf("cannot get non executed commitments: %w", err)
-		}
-
-		if len(nonExecutedCommitments) > 0 {
-			bundlesToExecute, err := c.state.getBundles(nextStateSyncExecutionIdx, maxBundlesPerSprint)
-			if err != nil {
-				return fmt.Errorf("cannot get bundles: %w", err)
-			}
-
-			ff.commitmentsToVerifyBundles = nonExecutedCommitments
```

Comment:
Was it safe to remove this? Are we removing:
1. just bundles execution from client side or 
2. bundle verification as part of the consensus as well?
If it is 2., then I'd say we can easily remove `case *BundleProof` from the `fsm.VerifyStateTransactions` function. WDYT?

Reply:
It's 1, just bundle execution, we still maintain bundle verification in consensus. As commented in https://github.com/0xPolygon/polygon-edge/pull/964#discussion_r1033858701 removal of all bundle related things to come.